src/vmm/src/builder.rs

```diff
@@ -531,7 +531,9 @@ pub fn build_microvm(
     } else if let Some(cmdline) = &vm_resources.boot_config.kernel_cmdline_prolog {
         kernel_cmdline.insert_str(cmdline).unwrap();
     } else {
-        kernel_cmdline.insert_str(DEFAULT_KERNEL_CMDLINE).unwrap();
+        kernel_cmdline
+            .insert_str(format!("{DEFAULT_KERNEL_CMDLINE} rw"))
+            .unwrap();
```

Comment:
Since I took `rw` out of both architectures' `DEFAULT_KERNEL_CMDLINE`, I figured I had to put it back in here. Not sure how this is consumed though.